Documentation/JSON_Schemas/HeartBeat_Response.json

```diff
@@ -0,0 +1,106 @@
+{
+  "type": "object",
+  "properties": {
+    "user": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "integer"
+        },
+        "token": {
+          "type": "string"
+        },
+        "is_caesar": {
+          "type": "boolean"
+        }
+      },
+      "required": [
+        "id",
+        "token",
+        "is_caesar"
+      ]
+    },
+    "users": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "Status": {
+            "type": "string"
```

Comment:
Ah, I knew you we're gonna ask about this ambiguous line, the idea is to inform other clients of other players' status, eg "Selectring card", "Playing" or "Waiting", it's meant to be text coming directly from the server to be shown to the user.